### PR TITLE
Added Login/ Signup using firebase

### PR DESCRIPTION
**Pull request by vipulsnp**
This PR solves issue #25 

**What does this PR do?**
Added Login/ Signup using firebase.
**Any background context you want to provide?**
.env file should be created using template as in .env.example
**Screenshots?**

### CODE REVIEW

File: src/components/Home/WelcomeComponent.jsx [hunk 1]
```diff
@@ -1,8 +1,65 @@
...
+        Define Required  Scopes here
+    */
+
+    firebase.auth().signInWithPopup(provider).then((result) => {
+      //  Access Token for using Google API's
+      const token = result.credential.accessToken;
+      //  Signed-in user info.
+      const { user } = result;
+      // eslint-disable-next-line no-console
+      console.log("Token : ", token);
```
Comment (irshadjsr21) [inline src/components/Home/WelcomeComponent.jsx:23]: Don't log anything after login

File: src/components/Home/WelcomeComponent.jsx [hunk 2]
```diff
@@ -1,8 +1,65 @@
...
+      // eslint-disable-next-line no-console
+      console.log("User : ", user);
+
+      /*
+          Redirect to the required page here
```
Comment (irshadjsr21) [inline src/components/Home/WelcomeComponent.jsx:28]: Redirect to feed page after login/signup.

**Comment by vipulsnp** [comment]
@irshadjsr21  I am unable to figure out how to use React Context. Also logs will be removed once the data coming from firebase is used for further purposes.

**Comment by irshadjsr21** [comment]
@alizahir23 Can you add context in this PR?

**Comment by alizahir23** [comment]
Ok

**Pull request merged**